ext/native/ui/viewgroup.cpp

```diff
@@ -1314,8 +1314,10 @@ EventReturn ListView::OnItemCallback(int num, EventParams &e) {
 	View *focused = GetFocusedView();
 	OnChoice.Trigger(ev);
 	CreateAllItems();
-	if (focused)
-		SetFocusedView(e.v);
+	// TODO: At this point, focused may no longer exist, depending on what OnChoice.Trigger does.
+	// Disable the refocus feature for now.
+	// if (focused)
+	//	SetFocusedView(e.v);
```

Comment:
Hmm, this was introduced in de3c4.  Maybe we can use...

```c++
if (focused) {
	int offsetNum = num;
	for (int i = 0; i <= num; ++i) {
		if (hidden_.find(i) != hidden_.end()) {
			--offsetNum;
		}
	}

	SetFocusedView(linLayout_->GetViewByIndex(offsetNum));
```

-[Unknown]

Reply:
Actually, things seem to still work without it, at least in the cases I tried (change buffered rendered mode, etc). So for 1.5.2 I'm leaving it like this.

Reply:
Ah, in that case let's just remove it.  I suppose we never use a ListView that doesn't close on select?

-[Unknown]